app/facades/movies_facade.rb

```diff
@@ -19,8 +14,17 @@ def top_rated
  
     data = service.top_rated_20
 
-    results = data[:results].map do |data|
+    data[:results].map do |data|
       Movie.new(data)
     end
   end
+
+  def find_movie(movie_id)
```

Comment:
I really like how this is done, and will have to remember how to use this in the future.